language/en/common.php

```diff
@@ -128,6 +128,7 @@
 	'EDIT_CATEGORY'				=> 'Edit Category',
 	'ERROR'						=> 'Error',
 
+	'FEATURED_CONTRIBUTION'		=> 'Featured contribution',
```

Comment:
I'm just not sure about using the word "Featured". Since these are being selected totaly at random, they're not really _featured_, if that makes any sense. When I think of featured I think of us intentionally picking a contrib that we want to prominently feature, based on its merits. I'd almost prefer calling this something like:

"Random Contribution Of The Day" or maybe just "Contribution Of The Day" or if anybody has a better thought. 

Or maybe: "Recent Contribution of the Day" and "Historic Contribution of the Day"


Again my words suck but the germ of an idea is there I think.

Reply:
It might just be overcomplicating it if we change the wording. I was just thinking "featured" in the sense that it's featured prominently on the main page.